test/MultiSelleckt.tests.js

```diff
@@ -0,0 +1,530 @@
+'use strict';
+
+var MultiSelleckt = require('../lib/MultiSelleckt');
+
+var $ = require('jquery');
+var Mustache = require('Mustache');
+
+describe('MultiSelleckt', function(){
+    var multiSelleckt,
+        $el,
+        selectHtml  =
+            '<select multiple>' +
+                '<option value="1" selected>foo</option>' +
+                '<option value="2" data-meh="whee" data-bah="oink">bar</option>' +
+                '<option value="3" selected>baz</option>' +
+            '</select>',
+        mainTemplate =
+            '<div class="{{className}} custom" tabindex=1>' +
+                '<ul class="mySelections">' +
+                '{{#selections}}' +
+                '{{/selections}}' +
+                '</ul>' +
+                '<div class="selected">' +
+                    '<span class="selectedText">{{selectedItemText}}</span><i class="icon-arrow-down"></i>' +
+                '</div>' +
+                '<ul class="items">' +
+                    '{{#items}}' +
+                    '<li class="item" data-text="{{label}}" data-value="{{value}}">' +
+                        '{{label}}' +
+                    '</li>' +
+                    '{{/items}}' +
+                '</ul>' +
+            '</div>',
+        selectionTemplate =
+            '<li class="mySelectionItem custom-item" data-value="{{value}}">' +
+                '{{text}}<i class="icon-remove unselect"></i>' +
+            '</li>';
+
+    beforeEach(function(){
+        $el = $(selectHtml);
+    });
+
+    afterEach(function(){
+        $el = undefined;
+
+        if(multiSelleckt){
+            multiSelleckt.destroy();
+            multiSelleckt = undefined;
+        }
+    });
+
+    describe('initialization', function(){
+        describe('custom options', function(){
+            beforeEach(function(){
+                multiSelleckt = new MultiSelleckt({
+                    mainTemplate: mainTemplate,
+                    selectionTemplate: selectionTemplate,
+                    multiple: true,
+                    $selectEl : $el,
+                    className: 'selleckt',
+                    selectedTextClass: 'selectedText',
+                    selectionsClass: 'mySelections',
+                    selectionItemClass: 'mySelectionItem',
+                    placeholderText: 'click me!',
+                    alternatePlaceholder: 'click me again!',
+                    itemsClass: 'items',
+                    itemClass: 'item',
+                    unselectItemClass: 'unselectItem',
+                    selectedClass: 'isSelected',
+                    highlightClass: 'isHighlighted',
+                    showEmptyList: true
+                });
+            });
+
+            it('stores options.selectionTemplate as this.selectionTemplate',function(){
```

Comment:
I think the [same](https://github.com/BrandwatchLtd/selleckt/pull/65/files#r24995306) applies here `multiSelleckt.selectionTemplate === selectionTemplate`